src/main/java/com/itech/kinopoisk/service/impl/LoadServiceImpl.java

```diff
@@ -51,46 +31,35 @@ public void loadTop250Films() {
             System.out.println(e.getMessage());
         }
 
-        do {
-            pageNumber++;
+        RestTemplate restTemplate = new RestTemplate();
```

Comment:
Configure RestTemplate as @Bean and use:
@Autowired
private RestTemplate restTemplate;